src/pybind/mgr/dashboard/module.py

```diff
@@ -914,13 +954,24 @@ def perf(self, osd_id):
                     ceph_version=global_instance().version,
                     path_info='/osd' + cherrypy.request.path_info,
                     toplevel_data=json.dumps(toplevel_data, indent=2),
-                    content_data=json.dumps(self._osd(osd_id), indent=2)
+                    content_data=json.dumps(self.perf_data(osd_id), indent=2)
```

Comment:
Does this actually work?  `perf_data` is returning a completely different type (a cherrypy response) than `_osd` (a dict).

Reply:
This works, yes. Do you want me to change it?

Reply:
Ah, turns out that the cherrypy context managers work differently than I thought, and they don't actually change the type of the return value.  Never mind!